package/package.json

```diff
@@ -13,7 +14,8 @@
   "files": [
     "lib",
     "index.ts",
-    "CHANGELOG.md"
+    "CHANGELOG.md",
+    "tsconfig.json"
```

Comment:
I don't think we need to publish the `tsconfig.json`, do we? The `files` field tells NPM what should be included when the package is published.